ops/storage-ddls/cassandra-setup.cql

```diff
@@ -86,6 +86,7 @@ CREATE TABLE watchlist(
 CREATE TABLE blacklist(
     id uuid,
     conjunctivefilter frozen<list<text>>,
+    type text,
```

Comment:
The `type` keyword is reserved in Scala, so we may need to map it to something else when reading these records in. Alternately, we can name it `category`, `classification` or something like that.